src/Dev/Deprecation.php

```diff
@@ -58,12 +55,21 @@ class Deprecation
      * must be available before this to avoid infinite loops.
      *
      * @var boolean|null
+     * @deprecated 4.12.0 Will be removed without equivalent functionality to replace it
```

Comment:
This wasn't addressed?

Reply:
Have updated message to say `Use $is_enabled instead`

It's slightly better naming to use a new property because we're deprecated the old method `get_enabled()` and adding a new method `get_is_enabled()` - so the property matches the method.